runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -370,20 +370,22 @@
     @Getter
     private volatile boolean isShutdown = false;
 
-    @Getter
-    private static MetricRegistry defaultMetrics = new MetricRegistry();
+    @Deprecated
     @Getter
     @Setter
-    private MetricRegistry metrics = new MetricRegistry();
+    private MetricRegistry metrics = defaultMetrics;
 
-    /** Initialize a default static registry which through that different metrics
-     * can be registered and reported */
+    /**
+     * Initialize a default static registry which through that different metrics can be registered and reported
+     */
     static {
-        synchronized (defaultMetrics) {
-            MetricsUtils.metricsReportingSetup(defaultMetrics);
-        }
+        MetricsUtils.metricsReportingSetup();
     }
 
+    @Deprecated
+    @Getter
+    private static final MetricRegistry defaultMetrics = MetricsUtils.metrics;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)